py3status/modules/dropboxd_status.py

```diff
@@ -86,14 +86,16 @@ def dropbox(self):
 
         if status == "Dropbox isn't running!":
             color = self.py3.COLOR_BAD
-            status = self.status_off
+            _format = self.status_off
         elif status == "Up to date":
             color = self.py3.COLOR_GOOD
-            status = self.status_on
+            _format = self.status_on
         else:
             color = self.py3.COLOR_DEGRADED
             if self.status_busy is not None:
-                status = self.status_busy
+                _format = self.status_busy
+
+        status = self.py3.safe_format(_format, {'status': status})
```

Comment:
I was about to merge this but then doing a last review I think here we should just do
```
status = self.py3.safe_format(_format)
```

also `_format` can be undefined when `self.status_busy is None` so that needs fixing

@lasers sorry I updated the branch